edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.CloudProxy.Test/ClientTokenCloudConnectionTest.cs

```diff
@@ -439,7 +439,8 @@ IClient GetMockedDeviceClient()
                 TimeSpan.FromSeconds(20),
                 false,
                 Option.None<IWebProxy>(),
-                productInfoStore);
+                productInfoStore,
+                true);
```

Comment:
we should use true only for new tests recently added for nested edge scenario. right?

Reply:
the end goal is to only have the nested edge logic in product, at which point all the tests need to run in this configuration.  I've already previously updated these tests to mock out the new nested edge behavior, so might as well have them run through nested edge codepath now.